<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MantenimientoEqui</class>
 <widget class="QDialog" name="MantenimientoEqui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>351</width>
     <height>671</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Equipos</string>
   </property>
   <widget class="QLabel" name="lblFederacion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Federacion:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letPuntos">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblDirector">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Director Tecnico:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPartidosP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Partidos Pendientes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblGolesC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Goles en Contra:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letAusencias">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letGolesF">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letPartidosJ">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblGolesF">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Goles a Favor:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letDirector">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letPartidosP">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letFederacion">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="letGolesC">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblPuntos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Puntos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblGrupo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Grupo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAusencia">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ausencias:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letGrupo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblPartidosJ">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Partidos Jugados:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblJugadores">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jugadores</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtJugadores">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblEquipo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Equipo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="letEquipo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Opciones</string>
    </property>
    <widget class="QRadioButton" name="rdbModificar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Modificar</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbEliminar">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbInsertar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Insertar</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbInsertar_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pboAceptar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>710</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pboAceptar</sender>
   <signal>clicked()</signal>
   <receiver>MantenimientoEqui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>720</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
